language_server/language_server.cpp

```diff
@@ -76,12 +76,12 @@ auto LanguageServer::onReply(llvm::json::Value /*id*/,
   return true;
 }
 
-// Returns the text of first child of kind Parse::NodeKind::Name.
+// Returns the text of first child of kind Parse::NodeKind::IdentifierName.
 static auto GetName(const SharedValueStores& value_stores,
```

Comment:
Do you want to rename this to GetIdentifierName, or add a TODO that it should support names in general?

Reply:
Renamed. I *think* this shouldn't be looking for keyword names, given that it's used for symbol indexing.